### PR TITLE
CI: add basic Helm testing

### PR DESCRIPTION
**Pull request by adutra**
# Description

This PR introduces some basic CI testing for the Polaris Helm chart.

## Type of change

CI Tests.

# How Has This Been Tested?

CI.

# Checklist:

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: helm/polaris/Chart.yaml [hunk 1]
```diff
@@ -29,3 +29,6 @@ sources:
 keywords:
   - polaris
   - iceberg
+maintainers:
```
Comment (adutra) [inline helm/polaris/Chart.yaml:32]: This list is the result of `git shortlog -n -s -- helm`. It must contain valid GitHub handles.

File: .github/workflows/helm.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  helm-tests:
```
Comment (snazy) [inline .github/workflows/helm.yml:35]: Please add as a required check in `.asf.yaml`

**Review by snazy — COMMENTED** [review]
Overall LGTM - but add it as a required check?

**Review by snazy — APPROVED** [review]
LGTM! Thanks!

**Comment by flyrain** [comment]
Looks like all OSS PRs are blocked by the helm tests reporting as the following screenshot shows. Any ideas?

image

**Comment by adutra** [comment]
@flyrain yes, that's unfortunate, but @snazy already put up a PR to fix it: #374 – please help us get that one merged 🙏

**Comment by flyrain** [comment]
Cannot merge it from github due to the same blocker. We need a way to force push it.

**Pull request merged**